agent/chaldea/__init__.py

```
@@ -19,16 +19,17 @@
 logger = logging.getLogger(__name__)
 
 
-def fetch_and_convert(source: str, output_dir: Optional[str] = None) -> Optional[str]:
-    """
-    主入口编排: 通过 source 获取数据并生成 BBC 配置文件
+def fetch_share_data(source: str):
+    """下载 + 解码 Chaldea BattleShareData（BBC 与 auto-battle 共用入口）。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)